fails:Enumerator::Lazy#take_while calls the block with initial values when yield with multiple arguments
fails:Enumerator::Lazy#take_while when the returned lazy enumerator is evaluated by .force stops after specified times
fails:Enumerator::Lazy#take_while on a nested Lazy when the returned lazy enumerator is evaluated by .force stops after specified times
